src/components/AllocationsComponent.jsx

```diff
@@ -73,7 +97,7 @@ export class AllocationsComponent extends Component {
 
             <Table.Body>
               {
-                this.props.allAllocations.map((allocation) => {
+                (this.props.allAllocations[currentAllocations] || allocations).map((allocation) => {
```

Comment:
Why not declare a variable and assign `this.props.allAllocations[currentAllocations] || allocations` to it, the use it here?